### PR TITLE
Lesson 7

### PR DESCRIPTION
**Pull request by Mikalaitru**

### CODE REVIEW

File: lesson-7/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="HTML, CSS, JavaScript">
+    <meta name="author" content="Mikalai Trukhanovich">
+    <title>Mikalai Trukhanovich's Portfolio</title>
+</head>
+<style>
```
Comment (veneta13) [inline lesson-7/index.html]: Great job incorporating CSS into your portfolio! For best practices, the `<style>` tag should be nested within the `<head>` tag.

File: lesson-7/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    h1 {
+        color: red;
+    }
+
+    p {
+        color: blue;
+    }
+</style>
+
+<body>
```
Comment (veneta13) [inline lesson-7/index.html:24]: Please wrap the About, Experience and Connect sections within `<section>` elements and assign them appropriate `id`s (you can find an example in the homework assignment rubric).

**Review by veneta13 — CHANGES_REQUESTED** [review]
Overall, great job on your assignment! There are just a few adjustments needed - please review the separate comments I’ve left.

**Review by Mikalaitru — COMMENTED** [review]
Hi 
I made some changes
Thank you for your support

**Review by veneta13 — APPROVED** [review]
Great work!

**Pull request merged**